packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -243,6 +244,7 @@ export function getInternalReactConstants(
       SimpleMemoComponent: 15,
       SuspenseComponent: 13,
       SuspenseListComponent: 19, // Experimental
+      TracingMarkerComponent: 25, // Experimental
```

Comment:
Unfortunately I think this is wrong. We _should_ probably add a new version check above for the 18 RC (whichever one first introduces the tracing marker), but so long as this is additive only– we could avoid forking the whole block and just rely on the act that we should never see a type-of-work value of 25 for React 17. What do you think?